.ci/pytorch/win-build.sh

```diff
@@ -30,23 +30,67 @@ if [[ -n "$PYTORCH_FINAL_PACKAGE_DIR" ]]; then
     mkdir -p "$PYTORCH_FINAL_PACKAGE_DIR" || true
 fi
 
-# This directory is used only to hold "pytorch_env_restore.bat", called via "setup_pytorch_env.bat"
+# This directory is used only to hold "pytorch_env_restore.bat", called via "setup_pytorch_env.py"
 CI_SCRIPTS_DIR=$TMP_DIR/ci_scripts
 mkdir -p "$CI_SCRIPTS_DIR"
 
 if [ -n "$(ls "$CI_SCRIPTS_DIR"/*)" ]; then
     rm "$CI_SCRIPTS_DIR"/*
 fi
 
-export SCRIPT_HELPERS_DIR=$SCRIPT_PARENT_DIR/win-test-helpers
+export SCRIPT_HELPERS_DIR=${SCRIPT_PARENT_DIR}/win-test-helpers
```

Comment:
Curly braces shouldn't be needed here.

Reply:
Not needed. Other files use the curly brace pattern and this bring this line into that pattern.